spec/docx/smoke/api_rgb_color_spec.rb

```diff
@@ -6,4 +6,11 @@
     docx = builder.build_and_parse('js/docx/smoke/api_rgb_color/get_class_type.js')
     expect(docx.elements[1].nonempty_runs.first.text).to eq('Class Type = rgbColor')
   end
+
+  it 'ApiRGBColor | ToJSON method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_rgb_color/to_json.js')
+    first_paragraph_text = '{"rgba":{"red":0,"green":0,"blue":0,"alpha":255},"color":{"rgba":{"red":255,"green":111,"blue":61,"alpha":255},"type":"srgb"},"type":"uniColor"}'
```

Comment:
When reading data from a document, use JSON.parse() to get a ruby object and interact with it.